exercises/reverse-string/README.md

```diff
@@ -0,0 +1,10 @@
+# Reverse String
+
+Here, your task is to reverse a string. For example: input: "cool" output: "looc".
+Every character should be accounted for, including spaces. For example: input: "Hello World" output: "dlroW olleH"
+
+## Version compatibility
+This exercise has been tested on Julia versions >=1.2.0
```

Comment:
Did you write this file by hand or generate it automatically? It's missing the `## Source` section and it's tested on >=1.0, not >=1.2. You can find more information about generating it in the `CONTRIBUTING.md` and in [Generating a README](https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md#generating-a-readme). Let me know if any questions come up.